src/main/java/seedu/address/model/project/Project.java

```diff
@@ -81,6 +81,28 @@ public ParticipantList getParticipants() {
         return participants;
     }
 
+    /**
+     * Adds a participant to the {@code ParticipantList}.
+     *
+     * @param person {@code Person} to add.
+     */
+    public Project addParticipant(Person person) {
+        requireNonNull(person);
+        ParticipantList participants = this.participants.addParticipant(person);
```

Comment:
I've set the observable list to track any changes to the participant list, returning a new project might mess this up. I'll check on my end and change this to a mutable version if needed.